gnomad_qc/analyses/compute_pext.py

```diff
@@ -0,0 +1,279 @@
+"""Script to get release pext annotation files."""
+
+import argparse
+import logging
+from typing import Tuple
+
+import hail as hl
+from gnomad.resources import grch37 as gnomad_grch37
+from gnomad.resources import grch38 as gnomad_grch38
+from gnomad.utils.slack import slack_notifications
+from gnomad.utils.transcript_annotation import (
+    TISSUES_TO_EXCLUDE,
+    clean_tissue_name_for_browser,
+    create_tx_annotation_by_region,
+    perform_tx_annotation_pipeline,
+    summarize_transcript_expression,
+)
+from hail.utils import new_temp_file
+
+from gnomad_qc.analyses.resources import get_pext
+from gnomad_qc.resource_utils import (
+    PipelineResourceCollection,
+    PipelineStepResourceCollection,
+)
+from gnomad_qc.slack_creds import slack_token
+
+logging.basicConfig(
+    format="%(asctime)s (%(name)s %(lineno)s): %(message)s",
+    datefmt="%m/%d/%Y %I:%M:%S %p",
+)
+logger = logging.getLogger("pext")
+logger.setLevel(logging.INFO)
+
+
+def filter_to_test(
+    ht: hl.Table,
+    tx_mt: hl.MatrixTable,
+) -> Tuple[hl.Table, hl.MatrixTable]:
+    """
+    Filter `ht` and `tx_mt` to a test set of genes.
+
+    Genes to filter to:
```

Comment:
I'm not sure, I just copied Qin's gene list over for testing. To be honest, I didn't really use the tests, since I ran this in a notebook initially, and it doesn't take that long on the full dataset. If you have some genes that you think would be good to add for testing, I'm happy to add them

Reply:
ah that's helpful to know, thanks!

I don't have any genes that I regularly check pext for, so no suggestions here. not high priority, but maybe we could ask someone on the analysis or curation teams for genes they regularly check for future reference?